### ISSUE TITLE
Allow configurable controller z-stacking to bypass overlay objects

### ISSUE THREAD

**Issue by hakt0-r** [igrigorik/videospeed#690]
Sites like...
* primevideo.com
* iview.abc.net.au 
* sbs.com.au

have multiple layers separating their video from their overlay controls or other objects. This renders the use of mouse with VSC useless. 

Solution: 
A simple fix is to use the switch case statement and start adding exceptions. The **_smarter_** way would be to add a setting where we can play with the **z-Order** of the VSC-controller div tag, like a _bring-to-forward_ **_or_** _send-to-back_ one step/layer at a time, per site that is having issues and it then somehow remembers that setting. I am not _that_ skilled in JS sadly to implement the advanced solution, but I have implemented the easier version, at least for me.

**Comment by pepijnmm** [comment]
same problem with facebook: #693

**Comment by igrigorik** [comment]
> The **_smarter_** way would be to add a setting where we can play with the **z-Order** of the VSC-controller div tag, like a _bring-to-forward_ **_or_** _send-to-back_ one step/layer at a time, per site that is having issues and it then somehow remembers that setting.

I think this is the right way to solve this problem. The list of exceptions continues to grow and it is brittle and unmaintainable. I'll open a new issue for us to track this.

**Comment by pepijnmm** [comment]
I would say that implementing both options are good.
If I'm a new user it would be nice that websites that get used a lot like: youtube, facebook, netflix etc already work without setting up. Of course the question is when will a website be popular enough to get added to the list but a lot of users that use this extension don't know how to code so they won't make a pull request for a website that they use a lot.

There will then of course still be a lot of websites that won't work because of overlays, than the keyboard back and forth will be nice. Then when Facebook will update their website again videospeed users could still use the website with using the back and forth to fix it so they don't have to wait till the extension gets updated.

But that is just my opinion I understand that saving and keeping it updated is not something that you would want but in the last year I haven't seen a lot of changes made to fix the controller on overlays.

**Comment by igrigorik** [comment]
Yep, that's reasonable. We can commit the defaults for small set large and popular sites, but defer the rest to the user.
